<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_isim">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Kullanici Adi</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sifre">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sifre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_giris_yap">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Giris Yap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_kaydol">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaydol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_isim">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sifre">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
